### DISCUSSION TITLE
Menubar TypeError

### DISCUSSION THREAD

**Discussion by ctwice74** [primefaces/community#672] [PrimeVue]
When trying to do a Vite implementation of the router example of the Menubar component from [here](https://primevue.org/menubar/#router) I am getting this error:

>  Uncaught TypeError: (destructured parameter) is undefined

It is pointing to this line as the issue.
`
<a v-ripple :href="href" v-bind="props.action" @click="navigate">
  <span :class="item.icon" />
  <span class="ml-2">{{ item.label }}</span>
</a>
`
I am using 3.43.0

**Comment by tugcekucukoglu** [comment]
How can I see the problem?
